woocommerce/pinterest-for-woocommerce
# Fix product attributes css file

### Changes proposed in this Pull Request:

The `assets/product-attributes/index.css` file is not being generated by webpack. This is because of the default configuration in wp-scripts. To get around this I renamed the file to `style.scss` which generates the final file as `style-index.css`, this is the same naming convention used for the other css files in this extension.

### Detailed test instructions:
1. Clone this PR and run `npm install && npm run build` 
2. Confirm the file `assets/product-attributes/style-index.css` is generated
3. Confirm the same file is present in the generated zip file
4. Edit a variable product and confirm that the `product-attributes/style-index.css` is loaded in the browser and doesn't give a 404
5. View one of the variations to change the Pinterest attributes. The attributes box should have a border around it to group the attributes together

#### CSS file not loaded
![image](https://user-images.githubusercontent.com/11388669/149969074-2b994012-9724-4e38-9017-1f0cd0860cbc.png)

#### CSS file loaded
![image](https://user-images.githubusercontent.com/11388669/149969118-b96847ee-0424-40a0-820a-cc1cb3567133.png)


### Changelog entry
* Fix - Load styles for product attributes.
